packages/dds/tree/src/simple-tree/api/schemaCreationUtilities.ts

```diff
@@ -62,11 +62,10 @@ export function singletonSchema<TScope extends string, TName extends string | nu
 		NodeKind.Object,
 		SingletonNodeType,
 		Record<string, never>,
-		true
-	> &
-		(new (
-			data?: InternalTreeNode | Record<string, never>,
-		) => SingletonNodeType) = SingletonSchema;
+		true,
+		Record<string, never>,
+		undefined
+	> = SingletonSchema;
 
 	return toReturn;
```

Comment:
Not necessary for this PR, but the last comment line above just got me thinking - can't we do `return x satisfies X` rather than `const y: X = x; return y`? A little bit cleaner for the places we've used this pattern.

Reply:
`satisfies` does not impact the type (only errors if the type does not satisfy it), so it would not fix the noted issue and we would be back to having non deterministic API reports.

Reply:
I didn't notice this function was using an implicit/inferred return value - so yea never minde.